app/views/layouts/application.html.erb

```diff
@@ -12,18 +12,28 @@
   </head>
 
   <body>
-    <div class="container-fluid">
-      <%= render partial: 'header' %>
-
-      <% if flash[:alert] %>
-        <div class="alert"><%= flash[:alert] %></div>
-      <% end %>
+    <%= render partial: 'header' %>
 
-      <% if flash[:notice] %>
-        <div class="notice"><%= flash[:notice] %></div>
-      <% end %>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-12">
+          <% if flash[:alert] %>
+            <div class="alert alert-warning"><%= flash[:alert] %></div>
+          <% end %>
+          <% if flash[:notice] %>
+            <div class="alert alert-success"><%= flash[:notice] %></div>
+          <% end %>
+        </div>
+      </div>
 
       <%= yield %>
     </div>
+    <script type="text/javascript">
+      window.setTimeout(function() {
+        $(".alert").fadeTo(500, 0).slideUp(500, function(){
```

Comment:
Not entirely sure I agree that this makes for good UX. What if you don't immediately pay attention, you'll miss the entire alert (unless I misunderstand it?)

Reply:
It might get annoying to have to dismiss the alerts manually each time but if we feel strongly about it I have no problem removing this.

Reply:
They should be gone when you load the next page.